cpp/ps0001/ps0001soln.cpp

```diff
@@ -1,14 +1,18 @@
 #include <iostream>
-#include <limits.h>
+
 using namespace std;
 
 int main()
 {
-  int a;
-  int b;
-  int c;
-  cin >> a >> b >> c ;
-  int s = a + b + c ;
-  cout << s ;
+  // prompting users for 3 numbers to add 1/2
+  short num1;
+  short num2;
+  short num3;
+  cin >> num1 >> num2 >> num3;

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add